simulator-client/package-lock.json

```diff
@@ -222,9 +230,9 @@
       "integrity": "sha1-0XV+yaf7I3HYPPR1XOPifhCCk4g="
     },
     "node-fetch": {
-      "version": "2.6.6",
-      "resolved": "https://registry.npmjs.org/node-fetch/-/node-fetch-2.6.6.tgz",
-      "integrity": "sha512-Z8/6vRlTUChSdIgMa51jxQ4lrw/Jy5SOW10ObaA47/RElsAN2c5Pn8bTgFGWn/ibwzXTE8qwr1Yzx28vsecXEA==",
+      "version": "2.6.7",
+      "resolved": "https://registry.npmjs.org/node-fetch/-/node-fetch-2.6.7.tgz",
+      "integrity": "sha512-ZjMPFEfVx5j+y2yF35Kzx5sF7kDzxuDj6ziH4FFbOp87zKDZNx8yExJIb05OGF4Nlt9IHFIMBkRl41VdvcNdbQ==",
```

Comment:
I also received a warning about `node-fetch` for one of my hobby projects, albeit on a different major version.

```json
"node-fetch": {
      "version": "3.1.0",
      "resolved": "https://registry.npmjs.org/node-fetch/-/node-fetch-3.1.0.tgz",
      "integrity": "sha512-QU0WbIfMUjd5+MUzQOYhenAazakV7Irh1SGkWCsRzBwvm4fAhzEUaHMJ6QLP7gWT6WO9/oH2zhKMMGMuIrDyKw==",
      "version": "3.2.0",
      "resolved": "https://registry.npmjs.org/node-fetch/-/node-fetch-3.2.0.tgz",
      "integrity": "sha512-8xeimMwMItMw8hRrOl3C9/xzU49HV/yE6ORew/l+dxWimO5A4Ra8ld2rerlJvc/O7et5Z1zrWsPX43v1QBjCxw==",
```

![image](https://user-images.githubusercontent.com/43425812/150690181-422e74a4-0760-4b6d-97a2-e52df14d157f.png)

Reply:
okay ty